guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,705 @@
 {
```

Comment:
<p>Add note about server requirement</p><ul><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#arcgis.raster.analytics.build_multidimensional_transpose" target="_blank">build_multidimensional_transpose()</a></code></li><li class="ql-indent-1">Adds new parameter:</li></ul><p class="ql-indent-2"><code>delete_transpose (Parameter available in</code>ArcGIS Image Server 10.9 or higher<code>)</code></p><ul><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#summarize-raster-within" target="_blank">summarize_raster_within()</a></code></li><li class="ql-indent-1">Adds new parameter:</li></ul><p class="ql-indent-2"><code>percentile_interpolation_type (Parameter available in</code>ArcGIS Image Server 10.9 or higher<code>)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/820/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Note added!